doc/api/tls.md

```diff
@@ -2226,6 +2250,18 @@ added: v11.4.0
   `'TLSv1.3'`. If multiple of the options are provided, the lowest minimum is
   used.
 
+## `tls.DEFAULT_CIPHERS`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string} The default value of the `ciphers` option of
+  [`tls.createSecureContext()`][]. It can be assigned any of the supported
+  OpenSSL ciphers.  Defaults to the content of
+  `'crypto.constants.defaultCoreCipherList'`, unless changed using CLI options
```

Comment:
```suggestion
  `crypto.constants.defaultCoreCipherList`, unless changed using CLI options
```
I don't think we should wrap that in quotes as it implies a string, since we're pointing to the value/*contents* of that property here.

Reply:
Good catch, thanks. Not sure how the second set of quotes crept in there!